README.md

```diff
@@ -155,9 +163,10 @@ such as restricting discovered features with the --label-whitelist option._
 
 ### Network Features
 
-| Feature name   | Description                                                                         |
-| :------------: | :---------------------------------------------------------------------------------: |
-| [SRIOV][sriov] | Single Root Input/Output Virtualization (SR-IOV) enabled Network Interface Card
+| Feature | Attribute    | Description                                         |
+| ------- | ------------ | --------------------------------------------------- |
+| `sriov` | `capable`    | [Single Root Input/Output Virtualization][sriov] (SR-IOV) enabled Network Interface Card(s) present
```

Comment:
These names sriov, capable, configured look like code blocks in the table, I guess that wasn't the intended look? :)

Reply:
Fixed